tests/oss-fuzz/pdu_parse_target.c

```diff
@@ -4,6 +4,7 @@ int
 LLVMFuzzerTestOneInput(const uint8_t* data, size_t size) {
     coap_pdu_t *pdu = coap_pdu_init(0, 0, 0, size);
     if (pdu) {
+        coap_set_show_pdu_output(0);
         coap_pdu_parse(COAP_PROTO_UDP, data, size, pdu);
         coap_show_pdu(LOG_DEBUG, pdu);
```

Comment:
If you want the PDU to be show, you would need to set the log level appropriately.